pom.xml

```diff
@@ -48,17 +48,17 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.5</version>
+      <version>1.7.22</version>
     </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>1.0.13</version>
+      <version>1.1.9</version>
```

Comment:
Why not go for latest 1.2.1 while you are at it (or at least 1.1.10 which is the latest on the 1.1.x branch)?